functions/exercise14/script.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+function getAmountOfCoins(amount) {
+    var denominationsOfCoins = [25, 10, 5, 2, 1];
+    var amountOfCoins = [];
+    while (amount > 0) {
+        for (var i = 0; i < denominationsOfCoins.length; i++) {
+            var curentCoins = denominationsOfCoins[i];
+            if (amount >= curentCoins) {
+                amountOfCoins.push(curentCoins);
+                amount -= curentCoins;
+                i = -1;
```

Comment:
You did not refer to my comment:
"It is not good to change for loop index inside the loop.
Maybe you should consider using while loop here instead of foor loop?"
You still change for loop index inside the loop. 
Do you need two loops here?